packages/react-core/src/components/Toolbar/__tests__/ToolbarToggleGroup.test.tsx

```diff
@@ -124,8 +124,8 @@ describe('Toolbar', () => {
         <ToolbarContent>{items}</ToolbarContent>
       </Toolbar>
     );
-    expect(view.find('#save-button').exists());
-    expect(view.find('#clear-button').exists());
+    expect(screen.findByTestId('save-button')).toBeTruthy();
+    expect(screen.findByTestId('clear-button')).toBeTruthy();
```

Comment:
Unless there's an async need for using `findBy` queries  React Testing Library strongly leans towards using `getBy` queries for items that you expect to exist. It also prioritizes using "real" and accessible identifiers to select elements over using testIds, with `getByRole` using the `{ name: 'visible text' }` option as the ideal.

Link to the RTL docs where they talk about this: https://testing-library.com/docs/queries/about#types-of-queries

That being said when attempting to test using this query locally multiple of the button are being found, and I'm not sure why. If it's because of some async oddness I understand your use of `findBy`, but I see the duplication in the snapshot as well so I don't *think* that's the cause.

Additionally in my testing these tests seem to be false positives, as I can invalidate them by removing the testIds on the buttons and they still pass. I believe because `findBy` queries return promises.

Reply:
Gotcha, I think I went with find only because that was the syntax of the previous test framework. Also updated the expect so the test fails if the elements aren't found.